### PR TITLE
Voter hir output

### PR DESCRIPTION
**Pull request by Zeegomo**
Use VoterHIR as the output format for the snapshot tool.

Ready for review, but waiting for Splendid to deliver on the requested documentation to correctly parse representatives addresses

### CODE REVIEW

File: catalyst-toolbox/src/bin/cli/snapshot/mod.rs [hunk 1]
```diff
@@ -22,9 +25,19 @@ pub struct SnapshotCmd {
...
+
+    /// Voter group to assign representatives to.
+    /// If empty, defaults to "rep"
     #[structopt(short, long)]
-    discrimination: Discrimination,
+    representatives_group: Option<String>,
+
+    /// Url of the representative db api server
+    #[structopt(short, long)]
+    reps_db_api_url: String,
```
Comment (enzoc4) [inline catalyst-toolbox/src/bin/cli/snapshot/mod.rs:40]: what about using `Url` from the `url` crate?

File: catalyst-toolbox/src/snapshot/voting_group.rs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error(transparent)]
+    Io(#[from] reqwest::Error),
+}
+
+#[derive(GraphQLQuery)]
+#[graphql(
+    query_path = "resources/repsdb/all_representatives.graphql",
```
Comment (enzoc4) [inline catalyst-toolbox/src/snapshot/voting_group.rs:25]: are these files the things we are waiting for?

File: catalyst-toolbox/src/bin/cli/snapshot/mod.rs (hunk 1 above)
Comment (Zeegomo) [reply catalyst-toolbox/src/bin/cli/snapshot/mod.rs:40]: probably better 👍🏼

**Review by enzoc4 — APPROVED** [review]
this looks good, I don't really know about the address though

File: catalyst-toolbox/resources/repsdb/all_representatives.graphql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+  query AllReps {
+    representatives {
+    	data {
```
[the remark this answers is not in the document]
Comment (Zeegomo) [reply catalyst-toolbox/resources/repsdb/all_representatives.graphql]: Uh, there were mixed tabs and spaces, should have fixed it now

**Pull request merged**